### ISSUE TITLE
✨ Auto Command deactivate

### ISSUE THREAD

**Issue by frieza213** [MrKai77/Loop#230]
### Feature Request Description

So like suppose i was trying to move an app but i then decided not to do so. In such case i should have an option to automatically deactivate the loop if no action has been taken for a particular amount of time(let's say 1 second)

### Screenshots

![CleanShot 2024-02-24 at 23 50 58@2x](https://github.com/MrKai77/Loop/assets/121119257/a8a9eac0-dd96-4790-a4e9-0bb9f49a52e8)

### Additional Context

Screenshot app name is swish(a gesture based windows manager) works pretty well with loop

### Final Checks

- [X] My issue title is descriptive
- [X] This is a single feature request (multiple feature requests should be reported individually)

**Comment by JaceThings** [comment]
I'm curious - can you think of any situations where someone might actually want to keep the key pressed down for a while, instead of just letting go once they're done? I

It'd be helpful to know if there are any use cases we should keep in mind before we think about adding this auto-deactivation feature.

**Comment by frieza213** [comment]
Yes suppose I was gonna loop a window to the right half and then I decided it was better in that position so normally I'd press esc to exit the radial menu overlay or window, but that would take up my 2nd hand too and if I were to just take my finger off the keyboard it would loop to the right half that is why I suggest there should be an option to timeout the selection of window position such that the radial menu would disappear after a set time so I just have to use a finger instead of the two hands I was originally using to exit the loop(because I have mapped my activate loop key to right option so cannot use single hand to press right option and escape simultaneously)

**Comment by JaceThings** [comment]
Would using the Undo keybind work for this?
![001747@2x](https://github.com/user-attachments/assets/37f53c75-5a1f-4ce7-ba85-137af3d6506d)

**Comment by frieza213** [comment]
Extra keypress imo, and I don't find undo command reliable tbh

**Comment by SenpaiHunters** [comment]
I closed this issue because, it isn't something we'd likely add to Loop, or would be much benefit, but I may actually add this, however, it would be under a defaults setting if i actually do add it. Where you'd need to manually invoke it. I will keep it open for a bit, but this isn't really a top priority.

**Comment by frieza213** [comment]
Thanks for adding this feature — it’s niche and likely ignored or unused by many, but I really appreciate it. ❤️

**Comment by Maple38** [comment]
So... does anyone know where this option actually is? Might just be blind but I can't for the life of me find it. Not on 1.4.2, and not even the dev build.

**Comment by mrkai77** [comment]
@Maple38 it is accessible via terminal commands, as we didn't want to complicate the main UI too much. Instructions are attached in the pull request: https://github.com/mrkai77/Loop/pull/953

**Comment by Maple38** [comment]
So forgive me if I'm wrong, but wasn't the idea of the issue initially to _cancel_ the action on timeout? That's how it works in Swish, which was referenced at the top.

Something might have been lost in translation, as it appears the feature _applies_ the action.
